### PR TITLE
Conform PEP8

### PR DESCRIPTION
**Pull request by Sparks29032**
Any fixes beyond standard refactoring are commented below.

Also fixes issue with `if` statement in `pre-commit.yml`.

### CODE REVIEW

File: src/diffpy/pdfmorph/morphs/__init__.py [hunk 1]
```diff
@@ -28,6 +28,12 @@
 from diffpy.pdfmorph.morphs.morphsmear import MorphSmear
 from diffpy.pdfmorph.morphs.morphstretch import MorphStretch
 
+# List of morph base classes
+morph_base = [
+    Morph,
+    MorphChain,
+]
+
```
Comment (Sparks29032) [inline src/diffpy/pdfmorph/morphs/__init__.py:36]: Cannot remove these top-level imports.

File: src/diffpy/pdfmorph/tests/test_morph_func.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ def test_fixed_morph_with_morphfunc():
     y_morph = heaviside(x_target, lb * (1 + stretch), ub * (1 + stretch))
     cfg = morph_default_config(stretch=0.1)  # off init
     cfg["scale"] = 30
-    morph_rv = pdfmorph(
+    pdfmorph(
         x_morph,
         y_morph,
         x_target,
```
Comment (Sparks29032) [inline src/diffpy/pdfmorph/tests/test_morph_func.py:35]: Code only needs to be run to test, do not need to save the return values (`morph_rv`).

File: src/diffpy/pdfmorph/tests/test_morphshape.py [hunk 3]
```diff
@@ -6,14 +6,16 @@
 import numpy
 import pytest
 
+from diffpy.pdfmorph.morphs.morphishape import MorphISpheroid
+from diffpy.pdfmorph.morphs.morphshape import MorphSphere, MorphSpheroid
+
+# FIXME: add MorphISphere test
+
```
Comment (Sparks29032) [inline src/diffpy/pdfmorph/tests/test_morphshape.py:13]: Add FIXME after removing import. Theoretically we should require a test for `MorphISphere` as well.

File: src/diffpy/pdfmorph/__init__.py [hunk 4]
```diff
@@ -19,8 +19,11 @@
 # top-level import
 from diffpy.pdfmorph.pdfmorph_api import morph_default_config, pdfmorph, plot_morph
 
-# obtain version information
-from diffpy.pdfmorph.version import __version__
+top_level = [
+    pdfmorph,
+    plot_morph,
+    morph_default_config,
+]
```
Comment (Sparks29032) [inline src/diffpy/pdfmorph/__init__.py:26]: Cannot remove from `__init__.py`, added to top level. We can either add `# noqa` to the end of the import line or do this for top-level.

File: src/diffpy/pdfmorph/__init__.py [hunk 5]
```diff
@@ -17,10 +17,7 @@
 """
 
 # top-level import
-from diffpy.pdfmorph.pdfmorph_api import morph_default_config, pdfmorph, plot_morph
-
-# obtain version information
-from diffpy.pdfmorph.version import __version__
+from diffpy.pdfmorph.pdfmorph_api import morph_default_config, pdfmorph, plot_morph  # noqa: F401
```
Comment (Sparks29032) [inline src/diffpy/pdfmorph/__init__.py:20]: Ignore qa, this is not a `PEP8` violation.

File: src/diffpy/pdfmorph/morphs/__init__.py [hunk 6]
```diff
@@ -17,8 +17,8 @@
 """
 
 
-from diffpy.pdfmorph.morphs.morph import Morph
-from diffpy.pdfmorph.morphs.morphchain import MorphChain
+from diffpy.pdfmorph.morphs.morph import Morph  # noqa: F401
+from diffpy.pdfmorph.morphs.morphchain import MorphChain  # noqa: F401
```
Comment (Sparks29032) [inline src/diffpy/pdfmorph/morphs/__init__.py:21]: Ignore qa, this is not a `PEP8` violation in `__init__.py`.

File: .github/workflows/pre-commit.yml [hunk 7]
```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   pre-commit:
     # pull requests are a duplicate of a branch push if within the same repo.
-    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.repository
+    if: ${{ github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.repository }}
```
Comment (Sparks29032) [inline .github/workflows/pre-commit.yml:11]: Very important bracketing...

**Comment by sbillinge** [comment]
nice, thanks @Sparks29032

**Pull request merged**